blueprint/index.md

```diff
@@ -0,0 +1,153 @@
+
+* [Solution components](#solution-components "Goes to the Solutions components section")
+* [Software development kits (SDKs)](#software-development-kits-sdks "Goes to the Software development kits (SDKs) section")
+* [Prerequisites](#prerequisites "Goes to the Prerequisites section")
+* [Implementation steps](#implementation-steps "Goes to the Implementation steps section")
+* [Additional resources](#additional-resources "Goes to the Additional resources section")
+
+## Solution components
+
+* **Genesys Cloud** - A suite of Genesys cloud services for enterprise-grade communications, collaboration, and contact center management. You create and manage OAuth clients in Genesys Cloud.
+* **AWS SAM CLI** - A cross-platform CLI that provides a Lambda-like execution environment that lets you locally build, test, and debug applications defined by SAM templates.
+* **AWS DynamoDB** - A fully managed NoSQL database service that provides fast and predictable performance with seamless scalability.
+* **AWS Lambda** - A compute service that lets you run code without provisioning or managing servers.
+* **AWS EventBridge** - A serverless event bus that makes it easier to build event-driven applications at scale using events generated from your applications, integrated Software-as-a-Service (SaaS) applications, and AWS services.
+* **AWS CloudFormation** - A service that gives developers and businesses an easy way to create a collection of related AWS and third-party resources, and provision and manage them in an orderly and predictable fashion.
+* **Node.js** - An open-source, cross-platform JavaScript runtime environment.
+* **Python** - An interpreted high-level general-purpose programming language.
+
+## Software development kits (SDKs)
+
+* **AWS SDK for Python (Boto3)** - The AWS SDK for Python. Boto3 makes it easy to integrate your Python application, library, or script with AWS services including Amazon S3, Amazon EC2, Amazon DynamoDB. This blueprint uses Boto3 for writing user presence updates to DynamoDB.
+
+* **AWS SDK for JavaScript** - AWS SDK for JavaScript Client for Node.js, Browser and React Native. This blueprint uses the AWS SDK or JavaScript for writing user presence updates to DynamoDB.
+
+## Prerequisites
+
+### Specialized knowledge
+
+* Knowledge of AWS services including SAM, DynamoDB, Lambda, EventBridge and CloudFormation
+* Experience with TypeScript, JavaScript or Python
+
+### Genesys Cloud account
+
+* A Genesys Cloud license. For more information, see [Genesys Cloud pricing](https://www.genesys.com/pricing "Opens the Genesys Cloud pricing page") on the Genesys website.
+
+### Third-party software
+
+* Python installed. Recommended version: 3.9.5. For more information, see [Download Python](https://www.python.org/downloads/ "Opens the Download Python page") on the Python website.
```

Comment:
I'd change these from recommended version to instead simply indicate what versions were used in this project. We probably wouldn't recommend the exact same version in 6 months, but we're not going to keep this up to date every release of every dependency. Maybe something like "Python - this project was built using 3.9.5".